creator-node/src/serviceRegistry.js

```diff
@@ -40,6 +41,9 @@ class ServiceRegistry {
     await this.blacklistManager.blacklist(this.ipfs)
     const audiusLibs = (config.get('isUserMetadataNode')) ? null : await initAudiusLibs()
     this.libs = audiusLibs
+    // Initialize state machine
```

Comment:
nit: can we add `this.snapbackSM = null` to the constructor like we do w libs for consistency? makes it easy to see what services the registry affords just by looking at the constructor